src/Rules/StarterWeb/AI/OrganizationalAuth/Multiple/Common/project.json

```diff
@@ -36,6 +34,10 @@
     "Microsoft.AspNetCore.Server.IISIntegration.Tools": {
       "version": "__AspNetCorePlatformVersion__",
       "imports": "portable-net45+win8+dnxcore50"
+    },
+    "Microsoft.Extensions.SecretManager.Tools": {
```

Comment:
Does the SecretManager need to be registered in dependencies section such as razor-tooling?